pom.xml

```diff
@@ -1,8 +1,49 @@
 <project>
+
   <modelVersion>4.0.0</modelVersion>
   <groupId>software.amazon.ion</groupId>
   <artifactId>ion-java</artifactId>
-  <version>1.0.0</version>
+  <version>1.0.1-SNAPSHOT</version>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>
+    An Java implementation of the Amazon Ion data notation.
+  </description>
+  <url>https://github.com/amznlabs/ion-java/</url>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <developers>
+     <developer>
+       <name>Amazon Ion Team</name>
+       <email>ion-team@amazon.com</email>
+       <organization>Amazon.com Labs</organization>
+       <organizationUrl>https://github.com/amznlabs</organizationUrl>
```

Comment:
I don't think it matters--this is really just Maven Central metadata and I'd rather just keep it closer to the actual GitHub organization.